### PR TITLE
walkthrough update

### PR DESCRIPTION
**Pull request by maleck13**
updated ocm walkthough to take use from 1-2 clusters

### CODE REVIEW

**Comment by jasonmadigan** [comment]
Got as far as `kubectl label managedcluster kind-mctc-control-plane ingress-cluster=true`

but then got

`kubectl label managedcluster kind-mctc-control-plane ingress-cluster=true`

**Comment by jasonmadigan** [comment]
running `make local-setup OCM_SINGLE=true MCTC_WORKLOAD_CLUSTERS_COUNT=1`

at the end I get:

```
installing the hub cluster in kind-(mctc-control-plane)
./hack/local-setup.sh: line 224: clusteradm: command not found
make: *** [local-setup] Error 127
```

would've thought this would be installed via local-setup

**Comment by maleck13** [comment]
Yes it should be.  Try run 
```make clusteradm```

locally

**Comment by jasonmadigan** [comment]
OK looks like expects `multicluster-gateway-controller/bin` needs to be on your `$PATH` (the makefile could be changed to make this unnecessary)

**Review by jasonmadigan — CHANGES_REQUESTED** [review]

File: docs/ocm-control-plane-walkthrough.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 - Clone this repo locally 
-- run `make local-setup OCM_SINGLE=true MCTC_WORKLOAD_CLUSTERS_COUNT=1`
-- setup a `./controller-config.env` with the following key values
+- Setup a `./controller-config.env` file in the root of the repo with the following key values
 
 ```
     # this sets up your default managed zone
     AWS_DNS_PUBLIC_ZONE_ID=<AWS ZONE ID>
-    ZONE_ROOT_DOMAIN=some.domain.com
+    ZONE_ROOT_DOMAIN=<replace.this>
```
[the remark this answers is not in the document]
Comment (maleck13) [reply docs/ocm-control-plane-walkthrough.md:25]: no its actually the domain / subdomain that points to this zone

File: docs/ocm-control-plane-walkthrough.md [hunk 2]
```diff
@@ -235,30 +262,136 @@ spec:
...
-If you want go ahead and create the HTTPRoute in the second gateway cluster. Looking at the DNSRecord afterwards you will it has now got two endpoints.
+Give DNS a minute or two to update. You should then be able to execute
+
+```
+dig sub.replace.this 
+```
+and get back the correct A record. You should also be able to curl that endpoint
+
+```
+curl -k https://sub.replace.this
```
[the remark this answers is not in the document]
Comment (maleck13) [reply docs/ocm-control-plane-walkthrough.md:292]: you get anywhere with that ?

[the remark this answers is not in the document]
Comment (jasonmadigan) [reply docs/ocm-control-plane-walkthrough.md:292]: No, got stuck here for some reason

[the remark this answers is not in the document]
Comment (jasonmadigan) [reply docs/ocm-control-plane-walkthrough.md:292]: @maleck13 locally, with your setup, are you able to ping the GW IP from your host? i.e. `ping 172.32.0.1`

[the remark this answers is not in the document]
Comment (jasonmadigan) [reply docs/ocm-control-plane-walkthrough.md:292]: Resolved- https://github.com/chipmk/docker-mac-net-connect was not started. We should probably add a note about usage of this for macOS users.

[the remark this answers is not in the document]
Comment (maleck13) [reply docs/ocm-control-plane-walkthrough.md:294]: added

**Pull request merged**